docs/src/rules/no-restricted-imports.md

```diff
@@ -264,6 +275,46 @@ import pick from 'fooBar';
 import { isEmpty } from 'utils/collection-utils';
 ```
 
+::: incorrect { "sourceType": "module" }
+
+```js
+/*eslint no-restricted-imports: ["error", { patterns: [{
+    group: ["utils/*"],
+    importNamePattern: '^is',
+    message: "Use 'isEmpty' from lodash instead."
```

Comment:
```suggestion
    message: "Use 'is*' functions from lodash instead."
```

I think this message would make more sense for this example as the configuration doesn't disallow only "isEmpty" but any names that start with "is".